submissions/kuslosolodka/html-css-popup/style.css

```diff
@@ -0,0 +1,298 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Raleway", sans-serif;
+  text-decoration: none;
+}
+body {
+  overflow: hidden;
+}
+.popup-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 30px 0;
+}
+#popup-btn {
+  display: none;
+}
+#popup-btn:checked ~ .popup-menu {
+  display: block;
+}
+.popup-icon {
+  width: 50px;
+  height: 50px;
+}
+.popup-icon:hover {
+  box-shadow: rgb(85, 91, 255) 0px 0px 0px 3px, rgb(31, 193, 27) 0px 0px 0px 6px,
+    rgb(255, 217, 19) 0px 0px 0px 9px, rgb(255, 156, 85) 0px 0px 0px 12px,
+    rgb(255, 85, 85) 0px 0px 0px 15px;
+}
+.popup-menu {
+  margin: 10px 0 30px 570px;
+  box-shadow: 3px 6px 16px 0px rgba(184, 180, 184, 1);
+  height: 600px;
+  width: 400px;
+  overflow-y: scroll;
+  border-radius: 10px;
+  display: none;
+}
+.icons {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  margin: 30px 30px 30px 30px;
+}
+a {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)